### DISCUSSION TITLE
Clones not working - UMAX Supermac J700 - Power Computing Powerbase 180

### DISCUSSION THREAD

**Discussion by anti-matter** [BlueSCSI/BlueSCSI-v2#103] [Help] [answered]
Hi there!

Been trying to use the **external** BlueSCSIv2 with my two clones. On first boot, things seem to be working fine. However, when I go to install an OS on a BlueSCSIv2 drive, the drive unexpectedly unmounts and disappears as do other drives. When I put the card into my PC's card reader, I'm told it needs to be formatted before it can be used/read.

I tried with 2 different external BlueSCSIv2 devices with the same result. I'd attach a log, but the drive is inaccessible after the issue occurs.

Known issue? User error? How can I help debug?

**Comment by erichelgeson** [comment]
Hey @anti-matter 

Sorry you're having an issue - that does sound odd. I have 3 clones, a Motorola, PowerComputing, and Umax SuperMac E600 - and v2 works great on all of them.

Please first take a look at the [troubleshooting guide](https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Troubleshooting) in the wiki. Follow each step, dont skip any! If you're still having an issue please enable debug and share the log file.

**Reply by anti-matter** [reply, marked as answer]
Ok, I switched to a different SD card (same brand, different speed) and this time I formatted it without using the "quick format" option. That seems to have done the trick. All is good now.

**Reply by erichelgeson** [reply]
Great to hear! The SDFat library we use can be picky about SD card formatting sometimes.
